### PR TITLE
Initial Tailwind Commit

### PR DESCRIPTION
**Pull request by RishavOsaurus**

### CODE REVIEW

File: css/input.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```
Comment (SatyaRajAwasth1) [inline css/input.css]: I think its always suggested to have a new line at the end of file (EOF).

Would you consider adding it?

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5502
+}
```
Comment (SatyaRajAwasth1) [inline .vscode/settings.json]: Can we add this .vscode and other editor/ IDE related configs to .gitignore ?

File: html/index.html
Comment (SatyaRajAwasth1) [inline html/index.html:1]: What do you think on keeping this index/home file in the base directory and other html files inside the html directory ?

**Review by SatyaRajAwasth1 — COMMENTED** [review]
Please also create an issue and tag that issue on this commit.

Commit message like **"Initialized project skeleton"** would be great.

Verified!

File: html/index.html
Comment (RishavOsaurus) [reply html/index.html:1]: All the necessary changes were made

**Review by SatyaRajAwasth1 — APPROVED** [review]

**Pull request merged**